client/containers/DashboardSubmissions/ArbitrationMenu.tsx

```diff
@@ -16,6 +16,29 @@ type Props = {
 
 const ArbitrationMenu = (props: Props) => (
 	<div className="arbitration-menu-component">
+		{props.pub.submission.status === 'accepted' && (
```

Comment:
In the `PubReleaseDialog`'s current home on the Pub page, the button that opens it is disabled if there aren't any changes in the Pub draft since the last Release. We don't have any such check here (and it would be troublesome to pipe that data in). So you can use this new button to try creating a new, duplicate Release, but it will give the user an unhelpful error once it's been attempted.

To get around this for now, I propose only showing this button if the Pub has no Releases.